src/lib/Components/Bidding/Components/Margins.ts

```diff
@@ -0,0 +1,11 @@
+export const Margins = {
+  m1: {
+    margin: 10,
+  },
+  m2: {
+    margin: 20,
+  },
+  m3: {
+    margin: 30,
+  },
+}
```

Comment:
This is the thing I wanted to discuss that in in question. The idea is to extract margins/paddings that are always hard to generalize. The approach essentially follows [`styled-system`](https://github.com/jxnblk/styled-system) and [`basscss`](http://basscss.com/#basscss-margin), but I'm open to suggestions.

Reply:
Hmm, good question. It looks good to me at first glance, maybe we can get @sarahscott or @orta's input?

Reply:
Cool! Seems like doing it this way would let us easily integrate with the styled system that @l2succes and @damassi are working on?

Reply:
Yup, this is the idea -- move these things into a shared, external source of truth built upon `styled-system`.